packages/fuels-core/src/constants.rs

```diff
@@ -5,6 +5,8 @@ pub const DEFAULT_GAS_LIMIT: u64 = 1_000_000;
 pub const DEFAULT_GAS_PRICE: u64 = 0;
 pub const DEFAULT_BYTE_PRICE: u64 = 0;
 pub const DEFAULT_MATURITY: u32 = 0;
+pub const STRUCT_KEYWORD: &str = "struct ";
```

Comment:
Can we add a comment explaining this const? It's very unclear to unfamiliar readers why we have that whitespace at the end.